manifests/t/Tencent/DeskGo/3.2.1445.127/Tencent.DeskGo.locale.zh-CN.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Tencent.DeskGo
+PackageVersion: 3.2.1445.127
+PackageLocale: zh-CN
+Publisher: Tencent
+PackageName: 电脑管家-桌面整理安装程序

```

```suggestion
PackageName: 腾讯桌面整理
```
This must match Control Panel.